File: lectures/week01/3-Wednesday/homework/homework1.py
tip = 0.0

bill_total = 0.0

bill_original = float(input("Total bill amount?"))
service_level = input("Levle of service?")

              
if service_level == "bad":
    tip = (bill_original * 0.10)
elif service_level == "fair":
    tip = (bill_original * 0.15)
elif service_level == "good":
    tip = (bill_original * 0.20)

else:
    print("Try entering good, bad, or fair")
    
bill_total =  tip + bill_original
print("Tip = %.2f" % tip) 
print("Bill Total = %.2f" % bill_total)

File: lectures/week04/1-Monday/MY-LectureNotes/homework/homework1.js
function hello(name) {
  
    if(name == null)
    {
      console.log("Hello World")
    }
    else
    {
      console.log(`Hello ${name}`)
    }
}
  hello();

  function madlib(name, subject)
  {
    string = name + "'s favorite subject in school is " + subject;

    return string;
  }

  sentence = madlib('Anushka', 'art')

  console.log(sentence)

  function tipAmount(bill, svcLvl, pats)
  {
    tip = 0
    total = 0
    splitAmt = 0

    if(svcLvl == 'good')
    {
      tip = bill * .20
    }
    else if(svcLvl == 'fair')
    {
      tip = bill * .15
    }
    else if(svcLvl == 'bad')
    {
      tip = bill * .10
    }

    total = bill + tip
    
    splitAmt = total/pats
    return splitAmt
  }

  console.log(tipAmount(100, 'bad', 3))